test/t/test_make.py

```diff
@@ -45,3 +47,32 @@ def test_8(self, bash, completion):
     @pytest.mark.complete("make -", require_cmd=True)
     def test_9(self, completion):
         assert completion
+
+
+@pytest.mark.bashcomp(require_cmd=True, temp_cwd=True)
+class TestMake2:
+    def test_github_issue_544_1(self, bash):
+        print(os.getcwd())
+        print(bash.cwd)
```

Comment:
Debug remainders or something that should be removed, no?

Reply:
Yes, these are debug remnants.

Reply:
I have removed them, thank you.